### PR TITLE
Add script to kill zombie containers

### PR DESCRIPTION
**Pull request by SnirCm**
In order to test:
1. docker-compose up - start the agent
2. stop the agent
3. manually run only the minio container using his id, he should now be a zombie (try kill it should fail)
4. run the script and restart docker if on windows
5. all sca agent containers should be gone images should be intact 

in wsl you need to run it with '/bin/bash' - some linking they did to sh command make it fail 
tested on: wsl2-ubuntu, gitBash, Linux-ubuntu

### CODE REVIEW

**Comment by SnirCm** [comment]
[Create a script that cleans hanging docker images (like minio) in the agent](https://trello.com/c/pvRM1I8a/1056-create-a-script-that-cleans-hanging-docker-images-like-minio-in-the-agent)

**Review by dbolharovcmx — DISMISSED** [review]
Works for me. Good Job

File: remove-zombie-containers.sh [hunk 1]
```diff
@@ -0,0 +1,42 @@
+#!/bin/sh
+
+echo "Removing following containers:"
+docker ps -a --no-trunc -f name=sca-agent* --format "table {{.ID}}\t{{.Names}}" || exit
```
Comment (maor-rosenfeld) [inline remove-zombie-containers.sh:4]: If there are no running or hanging containers, just print a message and exit

File: remove-zombie-containers.sh [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+echo "-------------------------------------------"
+echo "Press any key to continue..."
+read -n 1
```
Comment (maor-rosenfeld) [inline remove-zombie-containers.sh:7]: This line blocks non-human executions. Add a `-y` flag to skip this.

File: remove-zombie-containers.sh [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+echo "Starting"
+ids=$(docker ps -a --no-trunc -f name=sca-agent* --format "{{.ID}}")
+
+# check if Linux or wsl/mingw by try find cmd.exe
+cmd=$(command -v cmd.exe || echo shell)
```
Comment (maor-rosenfeld) [inline remove-zombie-containers.sh:12]: Doesn't work for me with WSL2

```
$ ./remove-zombie-containers.sh
Removing following containers:
CONTAINER ID        NAMES
-------------------------------------------
Press any key to continue...
./remove-zombie-containers.sh: 7: read: Illegal option -n
Starting
Creating zombie container with cmd to access MobyLinuxVM
Unable to find image 'alpine:latest' locally
latest: Pulling from library/alpine
596ba82af5aa: Pull complete
Digest: sha256:d9a7354e3845ea8466bb00b22224d9116b183e594527fb5b6c3d30bc01a20378
Status: Downloaded newer image for alpine:latest
b35532833b8cbeb197f94f5267e5f724f2f1c8afe2d71b871850b94d409faead
docker: Error response from daemon: /dev/shm is not mounted, but must be for --ipc=host.
```

**Review by maor-rosenfeld — CHANGES_REQUESTED** [review]
Move this script and `show-scans-status.sh` to a `tools` folder.

File: remove-zombie-containers.sh (hunk 3 above)
Comment (SnirCm) [reply remove-zombie-containers.sh:12]: you need to run like this: /bin/bash remove-zombie-containers.sh
there is some linking in wsl to sh that make it fail

File: remove-zombie-containers.sh (hunk 2 above)
Comment (SnirCm) [reply remove-zombie-containers.sh]: Added -s | --silence

File: remove-zombie-containers.sh (hunk 1 above)
Comment (SnirCm) [reply remove-zombie-containers.sh]: done

File: remove-zombie-containers.sh (hunk 3 above)
Comment (SnirCm) [reply remove-zombie-containers.sh:45]: added it to the --help examples

Comment (maor-rosenfeld) [reply remove-zombie-containers.sh:45]: Just change the first line of the script to `/bin/bash`.

Comment (SnirCm) [reply remove-zombie-containers.sh:45]: it is only a problem on wsl2 but ok

**Review by maor-rosenfeld — APPROVED** [review]

**Pull request merged**